app/src/test/kotlin/me/myshows/android/api/BaseMockWebServerTest.kt

```diff
@@ -0,0 +1,29 @@
+package me.myshows.android.api
+
+import okhttp3.mockwebserver.Dispatcher
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import okhttp3.mockwebserver.RecordedRequest
+import org.junit.After
+import org.junit.Before
+
+abstract class BaseMockWebServerTest {
+
+    protected val server: MockWebServer = MockWebServer().apply {
```

Comment:
`server` can be `private`

Reply:
no, otherwise it can't be used by children of `BaseMockWebServerTest`